deploy/host-operator/appstudio-e2e/secrets.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: host-operator-secret
+type: Opaque
+data:
+  twilio.account.sid: QUM4MWE4MTcwMTQ5NDU2MmQ3OWFjZmRkNzlkNjhkZTg3Yw==
+  twilio.auth.token: NjZlMzllNGUxOTZiMjFlOTM1YzU0OTA4M2RhMGU3NzI=
+  twilio.from_number: KzE1MDA1NTUwMDA2
```

Comment:
This is minor but can we use some values which make it clear that those are dummy ones? The current values look real :)

Reply:
Yeah, I'll do that for all secrets in this repo - but in the next PR and in the next year :-)